### PR TITLE
create all routes and controller for items

### PR DESCRIPTION
**Pull request by sinabeeler**
@wu-alexander, can you have a look at the code and merge it? thx

### CODE REVIEW

File: app/controllers/items_controller.rb [hunk 1]
```diff
@@ -0,0 +1,46 @@
+class ItemsController < ApplicationController
+  before_action :set_item, only: [ :show, :edit, :update, :destroy ]
+
+  def index
```
Comment (wu-alexander) [inline app/controllers/items_controller.rb:4]: you might want to add @item = Item.all here

File: app/controllers/items_controller.rb [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    @item = Item.new
+  end
+
+  def create
+    @item = Item.new(item_params)
+    @item.user = current_user
+    if @item.save
+      redirect_to @item, notice: "Your item has been created"
+    else
+      render "new"
```
Comment (wu-alexander) [inline app/controllers/items_controller.rb:20]: i think this should be "render :new"

File: app/controllers/items_controller.rb [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+  def new
+    @item = Item.new
+  end
+
+  def create
+    @item = Item.new(item_params)
+    @item.user = current_user
+    if @item.save
+      redirect_to @item, notice: "Your item has been created"
```
Comment (wu-alexander) [inline app/controllers/items_controller.rb:18]: Think we can actually just delete the notice here

File: app/controllers/items_controller.rb [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+    else
+      render "new"
+    end
+  end
+
+  def edit
+  end
+
+  def update
+    @item.update(item_params)
```
Comment (wu-alexander) [inline app/controllers/items_controller.rb:28]: Can add if/else statement and "render :edit" like for create

**Comment by sinabeeler** [comment]
thx for checking. I changed everything and will merge it directly.

**Pull request merged**